gaphor/plugins/align/align.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import annotations
+
+from gaphor.abc import ActionProvider, Service
+from gaphor.i18n import gettext
+
+from gaphor.ui.event import DiagramSelectionChanged
+from gaphor.core import action, event_handler
+from gaphor.diagram.presentation import ElementPresentation
+
+from gaphor.transaction import Transaction
+
+
+class AlignService(Service, ActionProvider):
+    def __init__(self, event_manager, diagrams, tools_menu=None, dump_gv=False):
```

Comment:
I know that the pydot service uses the dump_gv parameter, is it needed for this service?

Reply:
Good catch, removed the unnecessary parts.